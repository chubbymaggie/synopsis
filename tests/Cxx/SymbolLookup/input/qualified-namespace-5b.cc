namespace A 
{
  struct x { };
  int x;
  int y;
}
namespace B 
{
  struct y {};
}
namespace C 
{
  using namespace A;
  using namespace B;
  int i = C::x; // ok, A::x (of type 'int')
  int j = C::y; // ambiguous, A::y or B::y
}

