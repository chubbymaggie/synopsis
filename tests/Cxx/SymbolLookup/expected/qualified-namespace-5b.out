Namespace : A
declared at line 1 in <input>
ClassSpec : x
declared at line 3 in <input>
Declarator : x
declared at line 4 in <input>
Declarator : y
declared at line 5 in <input>
Namespace : B
declared at line 7 in <input>
ClassSpec : y
declared at line 9 in <input>
Namespace : C
declared at line 11 in <input>
Name : A
declared at line 1 in <input>
Name : B
declared at line 7 in <input>
Declarator : i
declared at line 15 in <input>
Name : C::x
declared at line 4 in <input>
Declarator : j
declared at line 16 in <input>
Name : C::y
declared at line 5 in <input>
declared at line 9 in <input>
